source/blog/2020-11-11-using-paypal-with-solidus.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: Using PayPal with Solidus
+date: 2020-11-11
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/2020/09/02/solidus-paypal-commerce-platform/cover.jpg
+description: >
+  PayPal has been working with Solidus in a big way over the last year, find out what's new in the world of payment gateways, and which one makes more sense for your store.
+---
+
+We're excited to share some news with you. As you are probably aware, Solidus has been working with PayPal recently. They've been the gold sponsor for our last two conferences - and more recently, they've helped us revitalize Solidus PayPal Braintree and release the new Solidus PayPal Commerce Platform extension.
+
+We're excited to announce that Solidus PayPal Commerce Platform has just released version 0.2.1, and that Solidus PayPal Braintree has just released version 1.0. Both of these extensions are ready to go, all _you_ need to do is find out which one is right for you.
```

Comment:
`Solidus PayPal Braintree has just released version 1.0`

should be 

`Solidus PayPal Braintree has just released version 1.0.0`